src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2044,36 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    SendErrorToUi("Unable to find preset file: %s", file_exists.error().message());
+    return;
+  }
+
+#if defined(__linux)
+  const QString program{"dbus-send"};
+  const std::string message_content = absl::StrFormat(
+      "/org/freedesktop/FileManager1 org.freedesktop.FileManager1.ShowItems "
+      "array:string:\"file:///%s\" string:\"\"",
+      preset.file_path().string());
```

Comment:
Did you intend to make it one argument? I was expecting this to be split in 4 separate arguments? I could be wrong though.

Reply:
I misunderstood the syntax for the `dbus-send` command. Changed now. Thanks!